src/main/java/org/eclipse/golo/runtime/Module.java

```diff
@@ -12,16 +12,20 @@
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
 
-class Module {
+public final class Module {
```

Comment:
It's now used in a subpackage (`runtime.augmentation)`, and thus need to be public. Either that, or move up the objects from `runtime.augmentation`, but I didn't want to clutter the `runtime` package. Both have drawback.